app/pb_kits/playbook/pb_table/table.rb

```diff
@@ -2,94 +2,48 @@
 
 module Playbook
   module PbTable
-    class Table < Playbook::PbKit::Base
-      PROPS = %i[configured_aria
-                 configured_classname
-                 configured_container
-                 configured_dark
-                 configured_data
-                 configured_disable_hover
-                 configured_id
-                 configured_single_line
-                 configured_size
-                 configured_text
-                 block].freeze
-
-      def initialize(aria: default_configuration,
-                     classname: default_configuration,
-                     container: default_configuration,
-                     dark: default_configuration,
-                     data: default_configuration,
-                     disable_hover: default_configuration,
-                     id: default_configuration,
-                     single_line: default_configuration,
-                     size: default_configuration,
-                     &block)
-        self.configured_aria = aria
-        self.configured_classname = classname
-        self.configured_container = container
-        self.configured_dark = dark
-        self.configured_data = data
-        self.configured_disable_hover = disable_hover
-        self.configured_id = id
-        self.configured_single_line = single_line
-        self.configured_size = size
-        self.block = block_given? ? block : nil
+    class Table
+      include Playbook::Props
+
+      partial "pb_table/table"
+
+      prop :size, type: Playbook::Props::Enum,
+                  values: %w[sm md lg],
+                  default: "md"
+      prop :single_line, type: Playbook::Props::Boolean,
+                         default: false
+      prop :dark, type: Playbook::Props::Boolean,
+                  default: false
+      prop :disable_hover, type: Playbook::Props::Boolean,
+                           default: false
+      prop :container, type: Playbook::Props::Boolean,
+                       default: false
```

Comment:
Every story that uses the table kit that we bring in front of Kraig comes back to us with him wanting `container: true`. Can we set this as the default option, @jasoncypret ?

Reply:
As per the last conversation with Jason, the default for container has been set to true.